internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStoreGraveyard.java

```diff
@@ -201,6 +206,24 @@ public Store getStoreFromGraveyard(String clusterName, String storeName, Stat st
     return dataAccessor.get(path, stat, AccessOption.PERSISTENT);
   }
 
+  /**
+   * Get the creation time of a store in the graveyard.
+   * This returns the time when the store was put into the graveyard (via updateZNode).
+   * 
+   * @param clusterName the cluster name
+   * @param storeName the store name
+   * @return the creation time in milliseconds since epoch, or STORE_NOT_IN_GRAVEYARD if the store doesn't exist in graveyard
+   */
+  public long getStoreGraveyardCreationTime(String clusterName, String storeName) {
+    String path = getStoreGraveyardPath(clusterName, storeName);
+    Stat stat = dataAccessor.getStat(path, AccessOption.PERSISTENT);
+    if (stat == null) {
+      LOGGER.debug("Store {} does not exist in graveyard for cluster {}", storeName, clusterName);
+      return STORE_NOT_IN_GRAVEYARD;
+    }
+    return stat.getCtime();
```

Comment:
Should we use mtime instead of ctime? (If we remove store from graveyard upon creation then ctime is fine but if we don't then we should use mtime)

Reply:
I see, that makes sense.